hel_django_auditlog_extra/context.py

```diff
@@ -0,0 +1,90 @@
+import contextlib
```

Comment:
```suggestion
# This code was originally based on django-auditlog's set_actor context manager, see
# https://github.com/jazzband/django-auditlog/blob/v3.0.0/auditlog/context.py
import contextlib
```

Linking to the original source might be prudent here for additional context / history.